dashboard.csss-ci.com/server/api/controllers/rest/v1/Proposals.php

```diff
@@ -199,6 +230,9 @@ public function selected_put() {
         }
     }
 
+    /**
+     * Inserts work end date information into the proposals model database if all necessary fields are valid, otherwise throws an error  
```

Comment:
update not insert

updates a proposal with an end date to signify that the proposal is done